src/main/resources/entity-types/finance/drv_fund_type_details.json5

```diff
@@ -0,0 +1,46 @@
+{
+  id: 'b7e43005-3e3a-4633-82b9-a96fcd7d8c47',
+  name: 'drv_fund_type_details',
+  private: false,
+  fromClause: 'src_finance_fund_type as ftype',
+  columns: [
+    {
+      name: 'id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      isIdColumn: true,
+      valueGetter: "ftype.id"
+    },
+    {
+      name: '_version',
```

Comment:
Should this be kept as `_version` in here? I thought we were reserving `_` prefixed names in FQM

Reply:
I think we should drop the _, for the exact reason Noah said (`_` is reserved)